adafruit_circuitplayground/circuit_playground_base.py

```diff
@@ -518,11 +510,21 @@ def adjust_touch_threshold(self, adjustment):
               if cp.touch_A1:
                   print('Touched pad A1')
         """
-        for touch_in in self._touches:
+        for touch_in in self._touches.values():
             if isinstance(touch_in, touchio.TouchIn):
                 touch_in.threshold += adjustment
```

Comment:
I think the dict value is now always a `TouchIn`, so you don't need to check for that again.

Reply:
Oh good catch